scripts/start

```diff
@@ -1,3 +1,3 @@
 #!/bin/bash
 docker-compose down -v
-docker-compose up --build --force-recreate frontend
```

Comment:
I haven't encountered any problems since removing it so assume that it either wasn't needed or the problems we were seeing with bind mounted volumes have been resolved. Removing it make startup and tests much faster (obvs).